src/jestUtils.ts

```diff
@@ -60,6 +59,22 @@ import {
 import { State } from './State';
 import { hasProperty, withPrevAndCurrent } from './utils';
 
+// load fireEvent conditionally, so RNGH may be used in setups without testing-library
```

Comment:
Is this change needed in this PR?

Reply:
Not really, removed in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/af2e38b5d6a3996362bcbd9065fb8f54b2bdb894 in favor of https://github.com/software-mansion/react-native-gesture-handler/pull/1895.